fsharp-backend/tests/FuzzTests/ExecutePureFunctions.FuzzTests.fs

```diff
@@ -155,9 +158,12 @@ module Generators =
         | RT.TError ->
           let! msg = genExpr' RT.TStr size
           return callFn "Test" "typeError" 0 [ msg ]
+        | RT.TIncomplete ->
+          // WHATISTHIS does this seem reasonable, given the context?
+          let! returnType = G.RuntimeTypes.dType
+          return! genExpr' returnType size
```

Comment:
This function `exprFromType` is only called from `dvalFromType`, which is in turn used only by the `FnAndArgs` generator, to supply arguments for StdLib fns.

Given that context, does this seem a reasonable implementation for the rare case where a [random value](https://github.com/darklang/dark/blob/63057e05434fb26134ace13693443c69111a0e76/fsharp-backend/tests/FuzzTests/ExecutePureFunctions.FuzzTests.fs#L365) is to be generated for a fn param?

Reply:
In this context, we're returning `RT.Expr`s; there is no `EIncomplete`.

Since there is no `EIncomplete`, my best idea was to return `ENull`, `EBlank`, or something random. I figured "something random" would test the most.

Thoughts?

Reply:
Ah, I see. An `EBlank` is the Expr equivalent of `TIncomplete`